### ISSUE TITLE
Add data on hospitals at capacity and w/ hospital capacity numbers

### ISSUE THREAD

**Issue by enzoampil** [enzoampil/phcovid#17]
https://docs.google.com/spreadsheets/d/1Q7ZD5kkUI_hQHYzUWhGRWNY145FOosizWdZDwJLrhMA/edit?fbclid=IwAR1FZNzoEPpo3zzJDJn2-xOCXKPuzwC3eOPXnazyqF23FdiQeTH4twOPaZY#gid=1348578368

**Comment by mamerisawesome** [comment]
Data moved to

https://docs.google.com/spreadsheets/d/1l932R1sf732FQAkAxYNsX7osFDvU25scWnj_cpTqu6w/edit?fbclid=IwAR2P61pBNuW1FbtkU_XIa4ZS9aoQ2thEMVOtOKf5a-H5a0gCqDYH1LSmRbw#gid=320145592&fvid=512957077

**Comment by mamerisawesome** [comment]
Not sure how to move around though as these new sheets has a user experience that promotes filtering. We may face the previous issue with the DSPH dataset but can happen more frequently

**Comment by enzoampil** [comment]
Good point. My thought is we can initalize the function first despite that risk then worry about reliability later.
